solidity/contracts/system/KeepFactorySelection.sol

```diff
@@ -34,28 +38,38 @@ library KeepFactorySelection {
 
         KeepFactorySelector factorySelector;
 
-        // Standard ECDSA keep factory: KEEP stake and ETH bond.
+        // Standard ECDSA keep vendor and factory: KEEP stake and ETH bond.
         // Guaranteed to be set for initialized factory.
+        IBondedECDSAKeepVendor keepStakeVendor;
         IBondedECDSAKeepFactory keepStakeFactory;
 
-        // Fully backed ECDSA keep factory: ETH stake and ETH bond.
+        // Fully backed ECDSA keep vendor and factory: ETH stake and ETH bond.
```

Comment:
It's an _ETH-bond-only_ option to be precise.

Reply:
28c7611607a4765f597f074a7bf168476eb119c9